src/plugins/remote-cache/storage/s3.ts

```diff
@@ -2,25 +2,22 @@ import s3 from 's3-blob-store'
 import aws from 'aws-sdk'
 
 export interface S3Options {
-  accessKey?: string
-  secretKey?: string
-  region?: string
   endpoint?: string
   bucket: string
 }
-export function createS3({ accessKey, secretKey, bucket, region, endpoint }: S3Options) {
-  if (!accessKey || !secretKey || !(region || endpoint)) {
-    throw new Error(
-      `To use S3 storage "accessKey (S3_ACCESS_KEY)", "secretKey (S3_SECRET_KEY)", and one of "region (S3_REGION)" and "endpoint (S3_ENDPOINT)" parameters are mandatory.`,
-    )
+export function createS3({ bucket, endpoint }: S3Options) {
+  // Propagate "old" environment variables names to new ones for backwards compatibility
+  for (const [oldName, newName] of Object.entries({
```

Comment:
We can drop this since we will release this as major version bump.

Reply:
OK, I removed that 👍